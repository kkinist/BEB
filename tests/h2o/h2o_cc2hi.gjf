%chk=h2o.chk
%nprocs=4
%mem=1000mw
# CCSD(T,maxcyc=100)/GEN gfinput geom=check scf=xqc guess=check

BEB step 9: high-spin CCSD(T) for dication of h2o

2 3

H O 0
cc-pVTZ
****

