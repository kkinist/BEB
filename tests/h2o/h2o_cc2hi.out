 Entering Gaussian System, Link 0=g09
 Initial command:
 /usr/local/apps/g09/l1.exe "/wrk/irikura/Gau-30308.inp" -scrdir="/wrk/irikura/"
 Entering Link 1 = /usr/local/apps/g09/l1.exe PID=     30311.
  
 Copyright (c) 1988,1990,1992,1993,1995,1998,2003,2009,2013,
            Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 09 program.  It is based on
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 09, Revision D.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, B. Mennucci, 
 G. A. Petersson, H. Nakatsuji, M. Caricato, X. Li, H. P. Hratchian, 
 A. F. Izmaylov, J. Bloino, G. Zheng, J. L. Sonnenberg, M. Hada, 
 M. Ehara, K. Toyota, R. Fukuda, J. Hasegawa, M. Ishida, T. Nakajima, 
 Y. Honda, O. Kitao, H. Nakai, T. Vreven, J. A. Montgomery, Jr., 
 J. E. Peralta, F. Ogliaro, M. Bearpark, J. J. Heyd, E. Brothers, 
 K. N. Kudin, V. N. Staroverov, T. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. Rendell, J. C. Burant, S. S. Iyengar, J. Tomasi, 
 M. Cossi, N. Rega, J. M. Millam, M. Klene, J. E. Knox, J. B. Cross, 
 V. Bakken, C. Adamo, J. Jaramillo, R. Gomperts, R. E. Stratmann, 
 O. Yazyev, A. J. Austin, R. Cammi, C. Pomelli, J. W. Ochterski, 
 R. L. Martin, K. Morokuma, V. G. Zakrzewski, G. A. Voth, 
 P. Salvador, J. J. Dannenberg, S. Dapprich, A. D. Daniels, 
 O. Farkas, J. B. Foresman, J. V. Ortiz, J. Cioslowski, 
 and D. J. Fox, Gaussian, Inc., Wallingford CT, 2013.
 
 ******************************************
 Gaussian 09:  ES64L-G09RevD.01 24-Apr-2013
                17-May-2017 
 ******************************************
 %chk=h2o.chk
 %nprocs=4
 Will use up to    4 processors via shared memory.
 %mem=1000mw
 ---------------------------------------------------------------
 # CCSD(T,maxcyc=100)/GEN gfinput geom=check scf=xqc guess=check
 ---------------------------------------------------------------
 1/29=2,38=1/1;
 2/12=2,40=1/2;
 3/5=7,11=9,14=-4,16=1,24=10,25=1,30=1,116=-2/1,2,3;
 4/5=1/1;
 5/5=2,8=3,13=1,38=6/2,8;
 8/6=4,9=120000,10=1/1,4;
 9/5=7,6=100,14=2/13;
 6/7=2,8=2,9=2,10=2/1;
 99/5=1,9=1/99;
 -------------------------------------------------
 BEB step 9: high-spin CCSD(T) for dication of h2o
 -------------------------------------------------
 Structure from the checkpoint file:  "h2o.chk"
 Charge =  2 Multiplicity = 3
 Redundant internal coordinates found in file.
 O,0,0.,0.,-0.1405046397
 H,0,0.,-0.7615598004,0.4580938899
 H,0,0.,0.7615598004,0.4580938899
 Recover connectivity data from disk.
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          8           0        0.000000    0.000000   -0.140505
      2          1           0        0.000000   -0.761560    0.458094
      3          1           0        0.000000    0.761560    0.458094
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3
     1  O    0.000000
     2  H    0.968656   0.000000
     3  H    0.968656   1.523120   0.000000
 Stoichiometry    H2O(2+,3)
 Framework group  C2V[C2(O),SGV(H2)]
 Deg. of freedom     2
 Full point group                 C2V     NOp   4
 Largest Abelian subgroup         C2V     NOp   4
 Largest concise Abelian subgroup C2      NOp   2
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          8           0        0.000000    0.000000    0.119720
      2          1           0        0.000000    0.761560   -0.478879
      3          1           0        0.000000   -0.761560   -0.478879
 ---------------------------------------------------------------------
 Rotational constants (GHZ):    787.9091958    432.3087586    279.1468894
 General basis read from cards:  (5D, 7F)
 AO basis set in the form of general basis input (Overlap normalization):
      1 0
 S   7 1.00       0.000000000000
      0.1533000000D+05  0.5198089434D-03
      0.2299000000D+04  0.4020256215D-02
      0.5224000000D+03  0.2071282673D-01
      0.1473000000D+03  0.8101055358D-01
      0.4755000000D+02  0.2359629851D+00
      0.1676000000D+02  0.4426534455D+00
      0.6207000000D+01  0.3570644227D+00
 S   6 1.00       0.000000000000
      0.5224000000D+03 -0.4421150084D-04
      0.1473000000D+03 -0.1225910413D-02
      0.4755000000D+02 -0.1055177248D-01
      0.1676000000D+02 -0.6744526326D-01
      0.6207000000D+01 -0.1711986073D+00
      0.6882000000D+00  0.1073298561D+01
 S   1 1.00       0.000000000000
      0.1752000000D+01  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.2384000000D+00  0.1000000000D+01
 P   3 1.00       0.000000000000
      0.3446000000D+02  0.4116348957D-01
      0.7749000000D+01  0.2577628359D+00
      0.2280000000D+01  0.8024192744D+00
 P   1 1.00       0.000000000000
      0.7156000000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.2140000000D+00  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.2314000000D+01  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.6450000000D+00  0.1000000000D+01
 F   1 1.00       0.000000000000
      0.1428000000D+01  0.1000000000D+01
 ****
      2 0
 S   3 1.00       0.000000000000
      0.3387000000D+02  0.2549486323D-01
      0.5095000000D+01  0.1903627659D+00
      0.1159000000D+01  0.8521620222D+00
 S   1 1.00       0.000000000000
      0.3258000000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1027000000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.1407000000D+01  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.3880000000D+00  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.1057000000D+01  0.1000000000D+01
 ****
      3 0
 S   3 1.00       0.000000000000
      0.3387000000D+02  0.2549486323D-01
      0.5095000000D+01  0.1903627659D+00
      0.1159000000D+01  0.8521620222D+00
 S   1 1.00       0.000000000000
      0.3258000000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1027000000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.1407000000D+01  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.3880000000D+00  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.1057000000D+01  0.1000000000D+01
 ****

 There are    27 symmetry adapted cartesian basis functions of A1  symmetry.
 There are     7 symmetry adapted cartesian basis functions of A2  symmetry.
 There are    12 symmetry adapted cartesian basis functions of B1  symmetry.
 There are    19 symmetry adapted cartesian basis functions of B2  symmetry.
 There are    23 symmetry adapted basis functions of A1  symmetry.
 There are     7 symmetry adapted basis functions of A2  symmetry.
 There are    11 symmetry adapted basis functions of B1  symmetry.
 There are    17 symmetry adapted basis functions of B2  symmetry.
    58 basis functions,    86 primitive gaussians,    65 cartesian basis functions
     5 alpha electrons        3 beta electrons
       nuclear repulsion energy         9.0882413160 Hartrees.
 NAtoms=    3 NActive=    3 NUniq=    2 SFac= 2.25D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=    58 RedAO= T EigKep=  3.67D-03  NBF=    23     7    11    17
 NBsUse=    58 1.00D-06 EigRej= -1.00D+00 NBFU=    23     7    11    17
 Initial guess from the checkpoint file:  "h2o.chk"
 B after Tr=     0.000000    0.000000    0.000000
         Rot=    1.000000    0.000000    0.000000    0.000000 Ang=   0.00 deg.
 Initial guess orbital symmetries:
 Alpha Orbitals:
       Occupied  (A1) (A1) (B2) (B1) (A1)
       Virtual   (A1) (B2) (B2) (A1) (A1) (B1) (A1) (B2) (A2) (B1)
                 (A1) (B2) (B2) (A1) (B1) (A2) (B2) (A1) (A1) (A1)
                 (B2) (B1) (A1) (A2) (B2) (B1) (B2) (A1) (A2) (B1)
                 (B2) (A1) (A2) (A1) (B1) (B2) (B2) (A1) (A1) (B2)
                 (B1) (A1) (B1) (A2) (B1) (A1) (B2) (A1) (A2) (A1)
                 (B2) (B2) (A1)
 Beta  Orbitals:
       Occupied  (A1) (A1) (B2)
       Virtual   (A1) (B1) (A1) (B2) (B2) (A1) (A1) (A1) (B2) (B1)
                 (A2) (B1) (A1) (B2) (B2) (A1) (B1) (A2) (B2) (A1)
                 (A1) (A1) (B2) (B1) (A1) (A2) (B2) (B1) (B2) (A1)
                 (A2) (B1) (B2) (A1) (A2) (A1) (B2) (B1) (B2) (A1)
                 (A1) (B2) (B1) (A1) (B1) (A2) (A1) (B1) (B2) (A1)
                 (A2) (A1) (B2) (B2) (A1)
 Initial guess <Sx>= 0.0000 <Sy>= 0.0000 <Sz>= 1.0000 <S**2>= 2.0061 S= 1.0020
 Keep R1 and R2 ints in memory in symmetry-blocked form, NReq=3449867.
 Requested convergence on RMS density matrix=1.00D-08 within  64 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 SCF Done:  E(UHF) =  -74.7024301000     A.U. after   10 cycles
            NFock= 10  Conv=0.72D-08     -V/T= 1.9948
 <Sx>= 0.0000 <Sy>= 0.0000 <Sz>= 1.0000 <S**2>= 2.0099 S= 1.0033
 <L.S>= 0.000000000000E+00
 Annihilation of the first spin contaminant:
 S**2 before annihilation     2.0099,   after     2.0000
 QCSCF skips out because SCF is already converged.
 ExpMin= 1.03D-01 ExpMax= 1.53D+04 ExpMxC= 5.22D+02 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 HarFok:  IExCor=  205 AccDes= 0.00D+00 IRadAn=         5 IDoV=-2 UseB2=F ITyADJ=14
 ICtDFT= 12500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 Range of M.O.s used for correlation:     2    58
 NBasis=    58 NAE=     5 NBE=     3 NFC=     1 NFV=     0
 NROrb=     57 NOA=     4 NOB=     2 NVA=    53 NVB=    55
 Semi-Direct transformation.
 ModeAB=           4 MOrb=             4 LenV=    1048292079
 LASXX=        88542 LTotXX=       88542 LenRXX=      183223
 LTotAB=       94681 MaxLAS=      346788 LenRXY=           0
 NonZer=      271765 LenScr=      815104 LnRSAI=      346788
 LnScr1=     1039360 LExtra=           0 Total=      2384475
 MaxDsk=          -1 SrtSym=           T ITran=            4
 JobTyp=1 Pass  1:  I=   1 to   4.
 (rs|ai) integrals will be sorted in core.
 ModeAB=           4 MOrb=             2 LenV=    1048292079
 LASXX=        46128 LTotXX=       46128 LenRXX=       88695
 LTotAB=       42567 MaxLAS=      173394 LenRXY=           0
 NonZer=      134823 LenScr=      785920 LnRSAI=      173394
 LnScr1=      785920 LExtra=           0 Total=      1833929
 MaxDsk=          -1 SrtSym=           T ITran=            4
 JobTyp=2 Pass  1:  I=   1 to   2.
 (rs|ai) integrals will be sorted in core.
 Spin components of T(2) and E(2):
     alpha-alpha T2 =       0.4984673837D-02 E2=     -0.2949845339D-01
     alpha-beta  T2 =       0.1998665997D-01 E2=     -0.1073705042D+00
     beta-beta   T2 =       0.7944394450D-03 E2=     -0.3813376361D-02
 ANorm=    0.1012800954D+01
 E2 =    -0.1406823339D+00 EUMP2 =    -0.74843112433896D+02
 (S**2,0)=  0.20099D+01           (S**2,1)=  0.20025D+01
 E(PUHF)=      -0.74705739317D+02        E(PMP2)=      -0.74845181610D+02
 Keep R2 and R3 ints in memory in symmetry-blocked form, NReq=3411002.
 Iterations= 100 Convergence= 0.100D-06
 Iteration Nr.   1
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        24
 NAB=     8 NAA=     6 NBB=     1.
 E(PMP3)=      -0.74858964164D+02
 MP4(R+Q)=  0.18236307D-02
 E3=       -0.14655427D-01        EUMP3=      -0.74857767861D+02
 E4(DQ)=   -0.11259628D-02        UMP4(DQ)=   -0.74858893824D+02
 E4(SDQ)=  -0.16205798D-02        UMP4(SDQ)=  -0.74859388441D+02
 DE(Corr)= -0.15351735     E(Corr)=     -74.855947445    
 NORM(A)=   0.10162797D+01
 Iteration Nr.   2
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        24
 NAB=     8 NAA=     6 NBB=     1.
 DE(Corr)= -0.15653145     E(CORR)=     -74.858961551     Delta=-3.01D-03
 NORM(A)=   0.10174431D+01
 Iteration Nr.   3
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        24
 NAB=     8 NAA=     6 NBB=     1.
 DE(Corr)= -0.15742548     E(CORR)=     -74.859855575     Delta=-8.94D-04
 NORM(A)=   0.10177872D+01
 Iteration Nr.   4
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        24
 NAB=     8 NAA=     6 NBB=     1.
 DE(Corr)= -0.15768505     E(CORR)=     -74.860115152     Delta=-2.60D-04
 NORM(A)=   0.10178579D+01
 Iteration Nr.   5
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        24
 NAB=     8 NAA=     6 NBB=     1.
 DE(Corr)= -0.15771619     E(CORR)=     -74.860146292     Delta=-3.11D-05
 NORM(A)=   0.10178607D+01
 Iteration Nr.   6
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        24
 NAB=     8 NAA=     6 NBB=     1.
 DE(Corr)= -0.15771797     E(CORR)=     -74.860148067     Delta=-1.78D-06
 NORM(A)=   0.10178617D+01
 Iteration Nr.   7
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        24
 NAB=     8 NAA=     6 NBB=     1.
 DE(Corr)= -0.15771829     E(CORR)=     -74.860148385     Delta=-3.18D-07
 NORM(A)=   0.10178620D+01
 Iteration Nr.   8
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        24
 NAB=     8 NAA=     6 NBB=     1.
 DE(Corr)= -0.15771849     E(CORR)=     -74.860148586     Delta=-2.00D-07
 NORM(A)=   0.10178619D+01
 Iteration Nr.   9
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        24
 NAB=     8 NAA=     6 NBB=     1.
 DE(Corr)= -0.15771846     E(CORR)=     -74.860148557     Delta= 2.89D-08
 NORM(A)=   0.10178618D+01
 Largest amplitude= 3.23D-02
 Time for triples=        2.05 seconds.
 T4(CCSD)= -0.27731444D-02
 T5(CCSD)=  0.35954092D-04
 CCSD(T)= -0.74862885747D+02

 S**2, projected HF & approx projected MPn energies after annihilation of
 unwanted spin states (see manual for definitions):

 spins       (S**2,0) (S**2,1)    PUHF        PMP2        PMP3        PMP4
 annihilated
 s+1         1.99998  2.00001  -74.705739  -74.845182  -74.858964
 s+1,s+2     2.00000  2.00000  -74.705733  -74.845176  -74.858959

 Discarding MO integrals.

 **********************************************************************

            Population analysis using the SCF density.

 **********************************************************************

 Orbital symmetries:
 Alpha Orbitals:
       Occupied  (A1) (A1) (A1) (B2) (B1)
       Virtual   (A1) (B2) (B2) (A1) (A1) (B1) (A1) (B2) (A2) (B1)
                 (A1) (B2) (B2) (A1) (B1) (A2) (A1) (B2) (A1) (A1)
                 (B2) (B1) (A1) (A2) (B1) (B2) (B2) (A1) (A2) (B1)
                 (B2) (A1) (A2) (A1) (B1) (B2) (B2) (A1) (A1) (B2)
                 (B1) (A1) (B1) (A2) (B1) (A1) (B2) (A1) (A2) (A1)
                 (B2) (B2) (A1)
 Beta  Orbitals:
       Occupied  (A1) (A1) (B2)
       Virtual   (A1) (B1) (A1) (B2) (B2) (A1) (A1) (B1) (A1) (B2)
                 (A2) (B1) (A1) (B2) (B2) (A1) (B1) (A2) (B2) (A1)
                 (A1) (A1) (B2) (B1) (A1) (A2) (B2) (B1) (B2) (A1)
                 (A2) (B1) (B2) (A1) (A2) (A1) (B2) (B1) (B2) (A1)
                 (A1) (B2) (B1) (A1) (B1) (A2) (A1) (B1) (B2) (A1)
                 (A2) (A1) (B2) (B2) (A1)
 The electronic state is 3-B1.
 Alpha  occ. eigenvalues --  -21.79018  -2.51024  -1.79072  -1.77519  -1.72548
 Alpha virt. eigenvalues --   -0.49065  -0.42162  -0.16565  -0.12416  -0.10053
 Alpha virt. eigenvalues --   -0.06424   0.04444   0.09446   0.10773   0.19056
 Alpha virt. eigenvalues --    0.30182   0.48791   0.73000   0.73859   1.06564
 Alpha virt. eigenvalues --    1.10903   1.22314   1.23507   1.29696   1.70461
 Alpha virt. eigenvalues --    2.03841   2.46909   2.61127   2.69722   2.73358
 Alpha virt. eigenvalues --    2.79806   2.95911   3.05367   3.09634   3.14397
 Alpha virt. eigenvalues --    3.16779   3.17398   3.42161   3.45265   3.65793
 Alpha virt. eigenvalues --    3.69337   3.94212   4.13029   4.20410   4.47923
 Alpha virt. eigenvalues --    4.94102   5.46937   5.70431   5.76802   5.89216
 Alpha virt. eigenvalues --    5.89353   5.98852   6.00855   6.13554   6.34129
 Alpha virt. eigenvalues --    6.67222   7.18456  11.64400
  Beta  occ. eigenvalues --  -21.69464  -2.19955  -1.68600
  Beta virt. eigenvalues --   -0.87958  -0.79256  -0.45561  -0.38894  -0.14627
  Beta virt. eigenvalues --   -0.10349  -0.00185   0.07203   0.08185   0.11269
  Beta virt. eigenvalues --    0.12966   0.23432   0.38056   0.49098   0.76428
  Beta virt. eigenvalues --    0.78272   1.18798   1.19035   1.26806   1.30445
  Beta virt. eigenvalues --    1.39987   1.72360   2.07268   2.49014   2.63227
  Beta virt. eigenvalues --    2.71340   2.80621   2.86669   2.97446   3.05839
  Beta virt. eigenvalues --    3.10489   3.17603   3.19403   3.28618   3.43999
  Beta virt. eigenvalues --    3.47966   3.71023   3.73708   3.97473   4.19488
  Beta virt. eigenvalues --    4.29320   4.54012   5.02677   5.55672   5.87872
  Beta virt. eigenvalues --    5.89746   5.98178   5.98953   6.02520   6.15000
  Beta virt. eigenvalues --    6.20898   6.42832   6.75155   7.24825  11.70434
          Condensed to atoms (all electrons):
               1          2          3
     1  O    6.420734   0.295546   0.295546
     2  H    0.295546   0.210011  -0.011469
     3  H    0.295546  -0.011469   0.210011
          Atomic-Atomic Spin Densities.
               1          2          3
     1  O    2.035655  -0.008402  -0.008402
     2  H   -0.008402  -0.013938   0.012914
     3  H   -0.008402   0.012914  -0.013938
 Mulliken charges and spin densities:
               1          2
     1  O    0.988175   2.018852
     2  H    0.505913  -0.009426
     3  H    0.505913  -0.009426
 Sum of Mulliken charges =   2.00000   2.00000
 Mulliken charges and spin densities with hydrogens summed into heavy atoms:
               1          2
     1  O    2.000000   2.000000
 Electronic spatial extent (au):  <R**2>=             10.3392
 Charge=              2.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=              0.0000    Y=              0.0000    Z=             -1.8706  Tot=              1.8706
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=             -3.5976   YY=             -0.2802   ZZ=             -1.7036
   XY=              0.0000   XZ=              0.0000   YZ=              0.0000
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=             -1.7371   YY=              1.5803   ZZ=              0.1569
   XY=              0.0000   XZ=              0.0000   YZ=              0.0000
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=              0.0000  YYY=              0.0000  ZZZ=             -1.2013  XYY=              0.0000
  XXY=              0.0000  XXZ=             -0.1254  XZZ=              0.0000  YZZ=              0.0000
  YYZ=             -2.2178  XYZ=              0.0000
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=             -1.6736 YYYY=             -0.6215 ZZZZ=             -1.6815 XXXY=              0.0000
 XXXZ=              0.0000 YYYX=              0.0000 YYYZ=              0.0000 ZZZX=              0.0000
 ZZZY=              0.0000 XXYY=             -0.8189 XXZZ=             -0.6355 YYZZ=              0.1976
 XXYZ=              0.0000 YYXZ=              0.0000 ZZXY=              0.0000
 N-N= 9.088241315969D+00 E-N=-1.875030067115D+02  KE= 7.509366942964D+01
 Symmetry A1   KE= 6.732758091892D+01
 Symmetry A2   KE= 5.979241054573D-33
 Symmetry B1   KE= 3.040793701566D+00
 Symmetry B2   KE= 4.725294809151D+00
 1\1\GINC-GAMBA\SP\UCCSD(T)-FC\Gen\H2O1(2+,3)\IRIKURA\17-May-2017\0\\# 
 CCSD(T,maxcyc=100)/GEN gfinput geom=check scf=xqc guess=check\\BEB ste
 p 9: high-spin CCSD(T) for dication of h2o\\2,3\O,0,0.,0.,-0.140504639
 7\H,0,0.,-0.7615598004,0.4580938899\H,0,0.,0.7615598004,0.4580938899\\
 Version=ES64L-G09RevD.01\State=3-B1\HF=-74.7024301\MP2=-74.8431124\MP3
 =-74.8577679\MP4D=-74.8607175\MP4DQ=-74.8588938\PUHF=-74.7057393\PMP2-
 0=-74.8451816\PMP3-0=-74.8589642\MP4SDQ=-74.8593884\CCSD=-74.8601486\C
 CSD(T)=-74.8628857\S2=2.00992\S2-1=2.002467\S2A=2.000037\RMSD=7.235e-0
 9\PG=C02V [C2(O1),SGV(H2)]\\@


 THERE'S NOTHING WRONG WITH FOLLOWING A TRAIN OF THOUGHT...
 AS LONG AS IT ULTIMATELY PULLS INTO THE STATION.
 Job cpu time:       0 days  0 hours  0 minutes 22.9 seconds.
 File lengths (MBytes):  RWF=     25 Int=      0 D2E=      0 Chk=      1 Scr=      1
 Normal termination of Gaussian 09 at Wed May 17 13:36:49 2017.
